source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -13,17 +13,227 @@ Document Data Format: POJOs
+  you omit the getter or setter methods for a property field, it is accessed
+  or assigned directly.
+
+
+Configure the Driver for POJOs
+------------------------------
+
+To set up the driver to store and retrieve POJOs, we need to specify:
+
+- The ``PojoCodecProvider``, a codec provider that includes
+  :doc:`Codecs </fundamentals/data-formats/codecs>` that define how to
+  encode/decode the data between the POJO and MongoDB document, and which
+  POJO classes or packages that the Codecs should apply to.
+- A :java-docs:`CodecRegistry <apidocs/bson/org/bson/codecs/configuration/CodecRegistry.html>`
+  instance that contains the codec provider and other related information.
+- A ``MongoClient``, ``MongoDatabase``, or ``MongoCollection`` instance
+  configured to use the ``CodecRegistry``.
+- A ``MongoCollection`` instance configured to cast to your POJO document
+  class.
+
+Follow the steps below to see how to perform each of the configuration
+requirements:
+
+
+1. Configure the :java-docs:`PojoCodecProvider </apidocs/bson/org/bson/codecs/pojo/PojoCodecProvider.html>`,
+   a codec provider that provides the Codecs that define how to encode the data
+   and on which classes to apply the Codecs. In this example, we use the
+   :java-docs:`automatic(true) </apidocs/bson/org/bson/codecs/pojo/PojoCodecProvider.Builder.html#automatic(boolean)>`
+   setting of the :java-docs:`PojoCodecProvider.Builder </apidocs/bson/org/bson/codecs/pojo/PojoCodecProvider.Builder.html>`
+   to apply the Codecs to any class and its properties.
+
+   .. code-block:: java
+
+      CodecProvider pojoCodecProvider = PojoCodecProvider.builder().automatic(true).build();
+
+.. note::
+
+   Codec providers also contain other objects such as ``ClassModel`` and
+   ``Convention`` instances that further define serialization behavior.
+   For more information on codec providers and customization, see our guide
+   on :doc:`POJO Customization </fundamentals/data-formats/pojo-customization>`.
+
+
+#. Add the ``PojoCodecProvider`` instance to a ``CodecRegistry``. The
+   ``CodecRegistry`` allows you to specify one or more codec providers to
+   encode the POJO data. In this example, we call the following methods:
+
+   - :java-docs:`fromRegistries() </apidocs/bson/org/bson/codecs/configuration/CodecRegistries.html#fromRegistries(org.bson.codecs.configuration.CodecRegistry...)>`
+     to combine multiple ``CodecRegistry`` instances into a single one
+   - :java-docs:`getDefaultCodecRegistry() <apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html?is-external=true#getDefaultCodecRegistry()>`
+     to retrieve a CodecRegistry instance from a list of codec providers
+   - :java-docs:`fromProviders() <apidocs/bson/org/bson/codecs/configuration/CodecRegistries.html#fromProviders(org.bson.codecs.configuration.CodecProvider...)>`
+     to create a ``CodecRegistry`` instance from the ``PojoCodecProvider``.
```

Comment:
nit: bullet point consistency, should either remove the period at the end of the bullet point or add one to the other two bullet points.

Reply:
Whoops, missed that, thanks.